### PR TITLE
Update WebNN ops impl status and add ChromeOS platform

### PR DESCRIPTION
**Pull request by ibelem**
1. Updated WebNN implementation status for WPT, XNNPACK, DirectML
2. Added WebNN implementation status on ChromeOS CPU backend (MLService)
3. Split the tables into two 
- Chromium implementation status
- JS ML Framework implementation status

10 239 115 52_4000_webnn-status_

@anssiko PTAL

### CODE REVIEW

File: webnn-status.md [hunk 1]
```diff
@@ -335,6 +588,52 @@ The total number of WebNN ops is 78. This table currently lists ops that are imp
             dml = `<td></td><td>${s.dml_chromium_version_added}</td>`;
           }
 
+          let mlservice = '';
+          if (s.mlservice_op.toString().trim() === "") {
```
Comment (anssiko) [inline webnn-status.md:598]: My local test build stops on this line with an error:

`typeError: Cannot read properties of undefined (reading 'toString')`

And this is how `s` looks like just prior to the error:

```
{
    "op": "argMax",
    "op_id": "argminmax",
    "version": "",
    "wpt": "arg_min_max",
    "wpt_progress": 4,
    "xnnpack_op": [
        ""
    ],
    "xnnpack_progress": 2,
    "xnnpack_chromium_version_added": "",
    "dml_op": [
        "REDUCE_FUNCTION_ARGMAX"
    ],
    "dml_progress": 4,
    "dml_chromium_version_added": "M122",
    "tflite_op": [
        ""
    ],
    "tflite_progress": 2,
    "tflite_version_added": "",
    "ort_op": [
        "ArgMax"
    ],
    "ort_progress": 4,
    "ort_version_added": "main",
    "notes": ""
}
```
It's missing `mlservice_op`, thus error.

Can you double-check whether this is related to my local build or something you can reproduce too?

File: webnn-status.md [hunk 2]
```diff
@@ -445,11 +751,19 @@ The total number of WebNN ops is 78. This table currently lists ops that are imp
             <th id="wpt_total"></th>
             <th id="x_total" colspan="2"></th>
             <th id="dml_total" colspan="2"></th>
+            <th id="mlservice_total" colspan="2"></th>
+          </tr>
+        `;
+
+         oplistFramework = oplistFramework + `
+          <tr>
+            <th id="spec_total"></th>
```
Comment (anssiko) [inline webnn-status.md]: This should display "78 / 78, 100.0%". Currently "156 / 78, 200%".

The spec_total formula needs tweaking. 200% looks cool though  :smile: :rocket:

**Review by anssiko — CHANGES_REQUESTED** [review]
@ibelem much thanks for this update!

While testing locally I ran into a few issues. Can you check if you can reproduce? It may be just my local setup, and if so, I'll push this to production and fix my local env.

**Comment by ibelem** [comment]
@anssiko Please try to change fetch("https://webmachinelearning.github.io/assets/json/webnn_status.json") to fetch("../assets/json/webnn_status.json") if you want to test it locally, there are paths differentiation on github pages and local environment, so I cannot use the relative path directly for upstream code.

And please let me know if it can solve issues above.

**Comment by ibelem** [comment]
@anssiko Fixed the spec_total and json fetch path issue, PTAL

**Review by anssiko — APPROVED** [review]
LGTM, thank you!

**Pull request merged**